docker-compose.yml

```diff
@@ -34,6 +34,12 @@ services:
          composer install
          && ./vendor/bin/phpunit
          '
+  app:
```

Comment:
Hele, asi budu potřebovat vysvětlit, proč to funguje takhle. 

1) `app` jsou jakoby "testy" - tj. to co se spouští na travisu. Tam jsou součástí "dev" balíčky. A není tam mount. (to mi přijde OK)
2) lokální vývoj je `tests` (což nechápu) a chybí tam "dev" balíčky - nechápu proč? Mimo to první věc, kterou to udělá, když to spustíš, tak to zavolá "composer install" a nainstaluje to všechno včetně dev balíčků. 

Jak z toho ven? 

Za mě: 
* nahodit standardní strukturu (tj. dev/app)
* všude instalovat dev závislosti
* targety zahodit (byť je to cool, tady mi to smysl nedává)

Reply:
jo `--dev` je koukám v defaultu. Díky.

Ono `tests` je spuštění aplikace -> proběhnou všechny testy. To by si teda přejmenoval na `app` a `app-xdebug` ?

Já potřebuju service, která nespustí aplikace = "spustí testy", ale tu která bude sloužit jenom pro build. Ta by se teda podle tebe měla jmenovat `dev` ?

Nechtěl jsem dávat do "produkčního" image dev závilosti, tak jsem to udělal odděleně, ale on se stejně nikde nepoužívá, tak je to jedno.

Reply:
Trochu jsem to zpřeházel v `docker-compose.yml` a `Dockerfile`. Mrkni prosím jestli ti to přijde lepší. Díky .-)